CHANGELOG-fix-collection-hook-deps.md

```diff
@@ -0,0 +1 @@
+- Change dependencies in useDatasetsCollections hooks to fix infinite loop.
```

Comment:
Did you change any dependencies?

Reply:
Sorry, I changed the variables in the `useMemo` hook's dependency array. I did not introduce any external dependencies here (like swr).